box.json

```diff
@@ -25,18 +25,29 @@
     "license":[
         {
             "type":"MIT",
-            "URL":""
+            "URL":"https://github.com/jcberquist/stripe-cfml/blob/master/LICENSE"
         }
     ],
-    "contributors":[],
-    "dependencies":{},
+    "contributors":[
+        "jcberquist",
+        "elpete"
+    ],
```

Comment:
Adding contributors sounds good to me, but then could you add @MordantWastrel and @sjdaniels? They have both contributed as well.